Quick/QuickSpec.m

```diff
@@ -101,7 +101,11 @@ + (SEL)addInstanceMethodForExample:(Example *)example {
         [example run];
     });
     const char *types = [[NSString stringWithFormat:@"%s%s%s", @encode(id), @encode(id), @encode(SEL)] UTF8String];
-    SEL selector = NSSelectorFromString(example.name.qck_selectorName);
+    NSString *selectorName = [NSString stringWithFormat:@"%@_%@_%ld",
+                              example.name.qck_selectorName,
+                              example.callsite.file,
+                              (long)example.callsite.line];
+    SEL selector = NSSelectorFromString(selectorName);
```

Comment:
New names are even worse, because the complete path is used:
`[Quick_OSXTests.FunctionalTests_AfterEachSpec executes_the_outer_afterEach_closures_once__but_not_before_this_closure__1__/<Path_to_your_project>/Quick/QuickTests/FunctionalTests/AfterEachTests.swift_22]`.

These get logged and are shown in the test navigator (but mostly only for a second, then Xcode shows something else... maybe another bug?).

What do you think would be a appropriate naming scheme? Another option would be, to check for `.isSharedExample == YES` to only mangle shared examples.